ngrinder-core/src/main/java/net/grinder/AgentController.java

```diff
@@ -144,18 +143,15 @@ public void run() {
 							connector = m_connectorFactory.create(agentConfig.getConnectionAgentPort());
 							occupyConnectionAgentSocket();
 						} else {
-							String controllerIP = getIP(agentConfig.getControllerHost());
-							agentConfig.setControllerIP(controllerIP);
-							connector = m_connectorFactory.create(controllerIP, agentConfig.getControllerPort());
+							connector = m_connectorFactory.create(agentConfig.getControllerIP(), agentConfig.getControllerPort());
```

Comment:
@donggyu04 

While removing the NetworkUtils constants, I found out this code block unnecessary.
This line changed in https://github.com/naver/ngrinder/pull/756. But, before entering to `AgentController#run`, `NGrinderAgentStarter` set up the controller IP address like below.
https://github.com/naver/ngrinder/blob/345f1d37bf380c5e788dbd28aa3508dd980a36bd/ngrinder-core/src/main/java/org/ngrinder/NGrinderAgentStarter.java#L137-L139

So, I think looking up the DNS again here is an unnecessary code block. Is it ok to remove these? 

Should I remove those lines on `NGrinderAgentStarter`? (If I remove those code lines on `NGrinderAgentStarter`, `AgentController` still check the DNS again.)

Reply:
@imbyungjun 

`NGrinderAgentStarter`'s code is called only once at agent startup.
but, this code exists for the case below

1. Agent connect to Controller(`www.mycontroller.com:10.10.10.10`) successfully
2. Controller shut down & restart with changed ip with same domain (`www.mycontroller.com:10.10.22.22`)
3. If remove this code block, agent will still retry connection to Controller with old ip(10.10.10.10) that saved when it startup

Reply:
Thx. I'll remove `NGrinderAgentStarter`'s controller IP setup code.